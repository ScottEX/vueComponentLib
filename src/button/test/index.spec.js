import { mount } from '@vue/test-utils';
import ComButton from '..';

test('render demo color', () => {
  const wrapper = mount(ComButton);
  expect(wrapper).toMatchSnapshot();
});
