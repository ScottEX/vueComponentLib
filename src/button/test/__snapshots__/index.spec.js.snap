// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render demo color 1`] = `
<button type="button" class="md-button default block">
  <div class="md-button-inner">
    <!---->
    <p class="md-button-content"></p>
  </div>
</button>
`;
